doc/source/installation.rst

```diff
@@ -1,24 +1,117 @@
+Quantum technologies, such as NISQ devices, are developed by research
+institutions and require a high level of knowledge of the physics and electronic
+devices used to prepare, execute and retrieve measurements from the experimental
+apparatus.
+
+In this context, Qibo proposes an agnostic approach to quantum simulation and
+hardware control, providing the required components and standards to quickly
+connect the classical hardware and experimental setup into a software stack
+which automates all aspects of a quantum computation.
+
+In the picture below, we summarize the major components of the Qibo "ecosystem".
+
+.. image:: overview.png
+
+The first component is the language API, based on Python 3, which defines the
+interface for the development of quantum applications, models and new
+algorithms. We also provide a large code-base of models and algorithms,
+presented with code examples and step-by-step tutorials. Finally, we provide
+several tools for the laboratory management and quantum hardware control.
+
+Qibo provides a plug and play mechanism of :ref:`backend drivers <backend-drivers>` which
+specializes the code for quantum simulation on different classical hardware
+configurations, such as multi-threading CPU, single GPU and multi-GPU, and
+similarly for quantum hardware control, from superconducting to ion trap
+technologies including FPGA and AWG devices.
+
+_______________________
+
+.. _backend-drivers:
+
+Backends drivers
+----------------
+
+As mentioned above, we provide backends for quantum simulation on classical
+hardware and quantum hardware management and control. In the image below we
+present a schematic view of the currently supported backends.
+
+.. image:: backends.png
+
+Quantum simulation is proposed through dedicated backends for single node
+multi-GPU and multi-threading CPU setups. Quantum hardware control is under
+development.
+
+_______________________
+
+Packages
+--------
+
+Following the overview description above, in this section we present the python
+packages for the modules and backends presented.
+
+Base package
+^^^^^^^^^^^^
+
+* :ref:`installing-qibo` is the base package for coding and using the API. This package contains all primitives and algorithms for start coding quantum circuits, adiabatic evolution and more (see :ref:`Components`). This package comes with a lightweight quantum simulator which works on multiple CPU architectures such as x86 and arm64.
+
+.. _simulation-backends:
+
+Simulation backends
+^^^^^^^^^^^^^^^^^^^
+
+We provide multiple simulation backends for Qibo, which are automatically loaded if the corresponding packages are installed, following the hierarchy below:
+
+* :ref:`installing-qibotf` (*recommended*): an efficient simulation backend for CPU, GPU and multi-GPU based on TensorFlow primitives. Install this package if you need to simulate quantum circuits with large number of qubits or complex quantum algorithms which may benefit from computing parallelism.
```

Comment:
I am not sure if "based on TensorFlow primitives" is correct here. I think it would be good to mention that `qibotf` uses custom operators for the multiplication of gates to states and some other operations to distinguish it from the pure `tensorflow` backend which uses only tensorflow primitives.

Reply:
Thanks, you are right.